tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -251,120 +252,145 @@ class ProjectUpdateTestTokenAuthenticated(APITestCase):
     def setUp(self):
         super().setUp()
         self.project = self.create_project(platform="javascript")
+        self.team = self.project.teams.first()
```

Comment:
nit: i think this is already set for you https://github.com/getsentry/sentry/blob/master/src/sentry/testutils/fixtures.py#L141-L143